### PR TITLE
[MPM] vtk output for particle conditions

### PR DESCRIPTION
**Pull request by VeronikaSinger**
Hi,

this adds the vtk output for particle conditions in MPM. It is very usefull to visualize the properties of the moving particle conditions during the simulation.

### CODE REVIEW

**Review by antonialarese — DISMISSED** [review]

File: applications/ParticleMechanicsApplication/python_scripts/particle_vtk_condition_output_process.py [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+
+
+def Factory(settings, Model):
+    if(type(settings) != KratosMultiphysics.Parameters):
+        raise Exception("Expected input shall be a Parameters object, encapsulating a json string")
+    model_part = Model[settings["Parameters"]["model_part_name"].GetString()]
+    parameters = settings["Parameters"]
+    return ParticleConditionVTKOutputProcess(model_part, parameters, Model)
+
+class ParticleConditionVTKOutputProcess(KratosMultiphysics.Process):
```
Comment (rubenzorrilla) [inline applications/ParticleMechanicsApplication/python_scripts/particle_vtk_condition_output_process.py]: This must derive from Kratos output process.

**Review by rubenzorrilla — COMMENTED** [review]
Why not deriving from the current VTK output process?

**Comment by VeronikaSinger** [comment]
Can someone reapprove please @KratosMultiphysics/mpm

**Review by lauramoremar — APPROVED** [review]

**Comment by VeronikaSinger** [comment]
thanks @laura-moreno

**Pull request merged**